applications/crossbar/priv/couchdb/views/rates.json

```diff
@@ -2,7 +2,7 @@
   ,"language":"javascript"
   ,"views":{
     "lookup": {
-      "map": "function(doc) { if (doc.deleted || doc.pvt_type !== 'rate') return; emit(parseInt(doc.prefix), null); }"
+      "map": "function(doc) { if (doc.deleted || doc.pvt_type !== 'rate') return; do_emit = function(direction) {emit([(''+doc.prefix), direction], {prefix: ''+doc.prefix, routes: (doc.routes || []).length ? doc.routes : ['^\\\\+?' + (''+doc.prefix).replace(/[^0-9]/g, '')], rate_name: doc.rate_name || 'prefix:'+doc.prefix}); }; if ((doc.direction || 'both') == 'both') { do_emit('inbound'); do_emit('outbound'); do_emit('any');} else {do_emit(doc.direction); do_emit('any');}}"
```

Comment:
I think JSON object keys must be enclosed by quotes. So should be `{\"prefix\": ''+doc.prefix,...`